modules/sdk-core/src/bitgo/tss/ecdsa/ecdsa.ts

```diff
@@ -82,6 +103,165 @@ export async function createCombinedKey(
   };
 }
 
+/**
+ * Creates the SignShare with User XShare and YShare Corresponding to BitGo
+ * @param {XShare} xShare User secret xShare
+ * @param {YShare} yShare YShare from Bitgo
+ * @returns {Promise<SignShare>}
+ */
+export async function createUserSignShare(xShare: XShare, yShare: YShare): Promise<SignShare> {
+  if (xShare.i === ShareKeyPosition.BITGO) {
```

Comment:
ShareKeyPosition.BITGO should be ShareKeyPosition.User

Reply:
Hmm, When user creates the signShare it should be the users xShare to be used for this?

Reply:
yes I believe the x share should belong to user for the user signing share. we should throw an error if it's not user's x share.

Reply:
Same point as below, considering backup also